lib/kino/explorer.ex

```diff
@@ -124,7 +124,18 @@ defmodule Kino.Explorer do
   end
 
   defp records_to_data(columns, records) do
-    Enum.map(columns, fn column -> Map.fetch!(records, column.key) |> Enum.map(&to_string/1) end)
+    Enum.map(columns, fn column ->
+      records |> Map.fetch!(column.key) |> Enum.map(&value_to_string/1)
+    end)
```

Comment:
In this case we have the types and we only need to check if printable if the type is binary!

Reply:
Good call! I also changed the columns to have the "binary" type, though it doesn't really matter on the Kino side, since we only use it for the icon and text is the default anyway.